pkg/controller/job/job_controller.go

```diff
@@ -1547,3 +1551,13 @@ func ensureJobConditionStatus(list []batch.JobCondition, cType batch.JobConditio
 	}
 	return list, false
 }
+
+func recordJobPodFinished(job *batch.Job, oldCounters batch.JobStatus) {
+	completionMode := completionModeStr(job)
+	if diff := job.Status.Succeeded - oldCounters.Succeeded; diff > 0 {
+		metrics.JobPodsFinished.WithLabelValues(completionMode, metrics.Succeeded).Add(float64(diff))
+	}
+	if diff := job.Status.Failed - oldCounters.Failed; diff > 0 {
+		metrics.JobPodsFinished.WithLabelValues(completionMode, metrics.Failed).Add(float64(diff))
+	}
```

Comment:
nit: we could directly `Add` the difference instead of checking that it is superior to zero.

Reply:
done